src/blog/2023/02/flowforge-1-4-0-released.md

```diff
@@ -0,0 +1,99 @@
+
+## New User Features
+
+
+**Automatic Device Provisioning**
+
+FlowForge 1.4 now includes automatic device onboarding for fleets of devices. Simply download a FlowForge device provisioning credential to a new device or fleet of devices. On device restart, the FlowForge agent and the Node-RED snapshot will automatically be provisioned to the device. [Issue #1212](https://github.com/flowforge/flowforge/issues/1212)
+
+
+<div><iframe width="560" height="315" src="https://www.youtube.com/embed/B7XK3TUklUU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+**Support for Staged Development**
+
+A new feature of 1.4 is the ability to setup staged deployments. This makes it possible to simply move a project between a Development > Test > Production for your Node-RED application delivery. [Issue #1580](https://github.com/flowforge/flowforge/issues/1580)
+
+
+<div><iframe width="560" height="315" src="https://www.youtube.com/embed/JRk-Cf7eNIo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+## Improvements
+
+- It is now possible to change the project type. This includes changing the version of Node-RED being used for the project and the small/medium/large.  [#595](https://github.com/flowforge/flowforge/issues/595)
+
+- Show a preview of Team Library entries on click. [#1657](https://github.com/flowforge/flowforge/issues/1657)
+
+- Add a synchronous mode for the FlowForge context store. [#17](https://github.com/flowforge/flowforge-nr-persistent-context/issues/17)
+
+- Add a Last Seen status for devices connecting to FlowForge. [#1599](https://github.com/flowforge/flowforge/issues/1599)
+
+- Added a check to ensure the team slug is unique. [#1609](https://github.com/flowforge/flowforge/issues/1609)
+
+- Optionally set snapshot as target at creation [#1527](https://github.com/flowforge/flowforge/issues/1527)
+
+- With FlowForge v1.4 some changes were made under the hood to speed up the recovery
+of Node-RED instances. On terminal failures of an instance it will now be
+automatically be redeployed with the correct flows. This uses Kubernetes features
+and is also available if you've installed through [kubernetes](https://flowforge.com/docs/install/kubernetes/).
+To migrate the old style of deployments to this system a restart or stack upgrade is needed.
+
+## Bug Fixes
+
+We've fixed the following bugs in this release.
+- Deleting your only team, doesn't exit from the team UI. [#1630](https://github.com/flowforge/flowforge/issues/1630)
+- Async Team Slug Check  [#1609](https://github.com/flowforge/flowforge/issues/1609)
+- Improve communication of Device Last Seen and Status [#1599](https://github.com/flowforge/flowforge/issues/1599)
+
+
+## Contributors
+
```

Comment:
We'd like the thank the following for their contributions to this release:

[UlisesGascon](https://github.com/UlisesGascon) for their work on [#74](https://github.com/flowforge/installer/pull/74)